### ISSUE TITLE
Complexity of later designs

### ISSUE THREAD

**Issue by mattbettcher** [emoon/68k_documentation_gen#3]
[68020UM](http://www.nxp.com/files/32bit/doc/ref_manual/MC68020UM.pdf) Read chapter 8.2 page 223 down until the tables (you have to read the intro to understand the tables). Once we start getting into the later designs ('020 and on) I feel the complexity of timing is crazy for some instructions (see the mov instruction) but still fairly simple for all the rest.

**Comment by emoon** [comment]
Yeah esp for 060 it gets even worse. There you a number of instructions can pair with others allowing them to be executed in the same cycle if conditions are correct. I really don't know how to deal with that so I started with 000 as it's the most basic one and was going to adjust as I would move to later gens.

**Comment by mattbettcher** [comment]
Sounds good.

**Comment by emoon** [comment]
I think in the move case we can perhaps just hard-code the values instead. Now it would of course be nicer to support it by calculating it but the ROI on doing something like that may not be worth it.

**Comment by mattbettcher** [comment]
I agree move is crazy complicated.
